cli_reference/openshift_developer_cli/debugging-applications-in-odo.adoc

```diff
@@ -0,0 +1,23 @@
+[id='debugging-applications-in-odo']
+= Debugging applications in `{odo-short}`
+include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: debugging-applications-in-odo
+toc::[]
+
+:FeatureName: Interactive debugging in `{odo-short}` 
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+With `{odo-short}`, you can attach a debugger to remotely debug your application. This feature is only supported for NodeJS and Java components.
+
+Components created with `{odo-title}` run in the debug mode by default.  A debugger agent runs on the component, on a specific port. To start debugging your application, you need to start port forwarding and attach the local debugger bundled in your IDE.
+
+include::modules/openshift-developer-cli-debugging-an-application.adoc[leveloffset=+1]
+include::modules/openshift-developer-cli-configuring-debugging-parameters.adoc[leveloffset=+1]
+
+
```

Comment:
Not critical, but good to clean up all these extra empty lines. If you use Atom as your editor, there is a setting that cleans these up upon saving.